.github/workflows/build+test+deploy.yml

```diff
@@ -1,27 +1,18 @@
-name: CI
+name: Build, Test and SelfCheck
 
 on:
-  pull_request:
-  workflow_dispatch:
-
-  # to execute once a day (more info see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule )
-  schedule:
-    - cron: '0 0 * * *'
```

Comment:
@xperiandri why remove the schedule? this way we make sure that CI is still working, and when it breaks for reasons unrelated to FSharpLint commits we notice right away. Before I added this schedule job it used to happen that suddenly some new commit landed in master and broke CI and we were scratching our heads for days wondering how that commit could break the build, until we realised the CI broke by itself, unrelated to the commit.